spec/config/routes.pending.yml

```diff
@@ -1,14 +1,7 @@
 ---
-AlertsListController:
-- class_icons
-- index
-- report_data
 AlertsMostRecentController:
```

Comment:
@akhilkr128 Do you know what this is?  Should it also be removed?

Reply:
I believe it is for the routing specs. These are getting fetched [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/c11dc20058f78c9460d79d0a56ce81513420af9a/spec/routes_spec.rb#L16) and testing it. I have removed the routes and controllers.

Reply:
I misunderstood the question, I will remove the `AlertsMostRecentController`.

Reply:
Made the changes.